augur/data/schema-export-v2.json

```diff
@@ -169,7 +169,8 @@
                         "properties": {
                             "key": {
                                 "description": "They key used to access the value of this coloring on each node",
-                                "type": "string"
+                                "type": "string",
+                                "not": {"enum": ["none"]}
```

Comment:
```suggestion
                               "not": {"const": "none"}
```

is another way to spell this, but I guess we might want to disallow other specific values in the future…